source/pyconform/modules/CLM_pft_to_CMIP6_vegtype.py

```diff
@@ -63,126 +63,126 @@ def __getitem__(self, index):
-
-	# Will contain weighted average for grass pfts on 2d grid
-	varo_vegType = np.zeros([len(time),len(lat),len(lon)]) 
+
+
+        # vegType = grass, shrub, or tree
+
+        # Tolerance check for weights summing to 1
+        eps = 1.e-5
+
+        # If 1, pft is active
+        active_pft = 1
+
+        # If 1, landunit is veg
+        veg_lunit  = 1
+
+        # C3 arctic grass,
+        # C3 non-arctic grass,
+        # C4 grass
+        beg_grass_pfts = 12
+        end_grass_pfts = 14
+
+        # broadleaf evergreen shrub - temperate,
+        # broadleaf deciduous shrub - temperate,
+        # broadleaf deciduous shrub - boreal
+        beg_shrub_pfts =  9
+        end_shrub_pfts = 11
+
+        # needleleaf evergreen tree - temperate,
+        # needleleaf evergreen tree - boreal,
+        # needleleaf deciduous tree - boreal,
+        # broadleaf evergreen tree - tropical,
+        # broadleaf evergreen tree - temperate,
+        # broadleaf deciduous tree - tropical,
+        # broadleaf deciduous tree - temperate,
+        # broadleaf deciduous tree - boreal
+        beg_tree_pfts  = 1
+        end_tree_pfts  = 8
+
+        # Will contain weighted average for grass pfts on 2d grid
+        varo_vegType = np.zeros([len(time),len(lat),len(lon)])
         tu = np.stack((pfts1d_lon,pfts1d_lat, pfts1d_active), axis=1)
 
-	ind = np.stack((grid1d_ixy,grid1d_jxy), axis=1)
-
-	lu = np.stack((land1d_lon,land1d_lat,land1d_ityplunit), axis=1)
-
-	# Loop over lat/lons
-	for ixy in range(len(lon)):
-	    for jxy in range(len(lat)):
-		grid_indx = -99
-		# 1d grid index
-		ind_comp = (ixy+1,jxy+1)
-		gi = np.where(np.all(ind==ind_comp, axis=1))[0]
-		if len(gi) > 0:
-		    grid_indx = gi[0] 
-
-		# Check for valid land gridcell
-		if grid_indx != -99:   
-
-		    # Gridcell lat/lons
-		    grid1d_lon_pt = grid1d_lon[grid_indx]
-		    grid1d_lat_pt = grid1d_lat[grid_indx]
-
-		    # veg landunit index for this gridcell
-		    t_var = (grid1d_lon_pt, grid1d_lat_pt, veg_lunit)
-		    landunit_indx = np.where(np.all(t_var == lu, axis=1))[0]
-
-		    # Check for valid veg landunit
-		    if landunit_indx.size > 0: 
-			if 'grass' in vegType:
-			    t_var = (grid1d_lon_pt,grid1d_lat_pt, active_pft)
-			    pft_indx = np.where( np.all(t_var == tu, axis=1) * (pfts1d_wtgcell > 0.) * (pfts1d_itype_veg >= beg_grass_pfts) * (pfts1d_itype_veg <= end_grass_pfts))[0]
-			elif 'shrub' in vegType:
-			    t_var = (grid1d_lon_pt,grid1d_lat_pt, active_pft)
-			    pft_indx = np.where( np.all(t_var==tu, axis=1) * (pfts1d_wtgcell > 0.) * (pfts1d_itype_veg >= beg_shrub_pfts) * (pfts1d_itype_veg <= end_shrub_pfts))[0]
-			elif 'tree' in vegType: 
-			    t_var = (grid1d_lon_pt,grid1d_lat_pt, active_pft)
-			    pft_indx = np.where( np.all(t_var==tu, axis=1) * (pfts1d_wtgcell > 0.) * (pfts1d_itype_veg >= beg_tree_pfts) * (pfts1d_itype_veg <= end_tree_pfts))[0]
-                          
-			# Check for valid pfts and compute weighted average
-			if pft_indx.size > 0:
```

Comment:
All of the tabs that were added to this file caused this error when called
`
TabError: inconsistent use of tabs and spaces in indentation
`
It might be best to revert all of these tabs because there is a now a mixture of spaces and tabs.

Reply:
Nevermind, I was looking at the wrong version.  These changes are okay.